### DISCUSSION TITLE
To emit a CSS abs() now: abs(#{100%})

### DISCUSSION THREAD

**Discussion by Jeko92** [twbs/bootstrap#39031] [Q&A] [answered]
I have tried to install new Bootstrap 5.3 Project with Webpack and I receive This error. Anyone else  already had such issue? How Can I resolve it?

**Comment by JamesVanWaza** [comment]
@Jeko92 please post the error you received

**Comment by julien-deramond** [comment, marked as answer]
This is linked to https://github.com/twbs/bootstrap/issues/39028. It'll be patched soon.
